docs/_articles/en/codebuilder/about.md

```diff
@@ -2,27 +2,20 @@
 title: Code Builder Overview
 lang: en
 ---
+
 ## Overview
+
 Salesforce Code Builder is a web-based integrated development environment that has all the power and flexibility of Visual Studio Code, Salesforce Extensions for VS Code, and Salesforce CLI in your web browser. Code Builder provides a modern developer experience for all developers, regardless of expertise level. Code Builder makes it easy for admins and developers alike to work in the cloud without having to worry about downloading software, setup or your machine specs.
```

Comment:
Code Builder makes it easy for admins and developers alike to work in the cloud without having to worry about downloading software, setup, or your machine specs.